nodejs/json-rpc-relay/integration.tests/mirror.node/eth.blocksHash.test.js

```diff
@@ -0,0 +1,74 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const Web3 = require('web3');
+const { curly } = require('node-libcurl');
+const { ethers } = require("ethers");
+const urlHandler = require('../../handlers/url.handler');
+const httpHeaderJson = ['Content-Type: application/json','Accept: application/json'];
+
+
+const blockHashTestnet = "0x5bacb48d5e59e753c1508f21dbe4c7b117cb2047b5e4d48be86decad9ec03e47231f22d16d241f6bab5529efdd971991";
+const blockHashMainnet = "0x0f1c15e5a19a04f1b03bcf7f8605d14f69c4d21356728486fce2ec929366b1bb5628a4eafb94342729176dea32e14a68";
+const hashTestnetParsed = blockHashTestnet.substring(0, 66);
+const hashMainnetParsed = blockHashMainnet.substring(0, 66);
```

Comment:
Cheeky part for Hedera, the hash payload from the api has be to be cut at 66 chars

Reply:
Good catch